backend/models/events.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
```

Comment:
What is this file for? Is it safe to delete now that we have a another file that defines the event model?